src/status_im2/contexts/onboarding/new_to_status/view.cljs

```diff
@@ -105,17 +106,22 @@
    [quo/page-nav
     {:align-mid?            true
      :mid-section           {:type :text-only :main-text ""}
-     :left-section          {:type                :blur-bg
-                             :icon                :i/arrow-left
-                             :icon-override-theme :dark
-                             :on-press            navigate-back}
-     :right-section-buttons [{:type                :blur-bg
-                              :icon                :i/info
-                              :icon-override-theme :dark
-                              :on-press            #(rf/dispatch
-                                                     [:show-bottom-sheet
-                                                      {:content getting-started-doc
-                                                       :shell?  true}])}]}]])
+     :left-section          {:type     :blur-bg
```

Comment:
I won't encourage using `on-long-press` in this issue as the user (Devs / Designers / QA) will eventually forget which button has this quick access.

IMO, Buttons are a visible UI element and are self-explanatory. The new (ugly - red-bordered) button is added for easy understanding that it's just for quick access. This will help a lot to quickly check and review newly built components in a PR.

Reply:
maybe we can have a particular icon or emoji - I don't like the red border 😆

Reply:
> maybe we can have a particular icon or emoji - I'm not a fan of the red border

Nice idea. I'm not a fan either. But the design team wants the UI elements which are not part of the design to be ugly.

How about this (little polished) ugly?

<img src="https://github.com/status-im/status-mobile/assets/19339952/bd0566dc-6d36-466e-aeed-9995da464c64" width="300" />

Reply:
I have removed the red border and updated the icon 👍